File: hw/aimacana/u2/hw16/python/taxes_lib/tax.py
class Tax:
    @staticmethod
    def compute_iva(amount: float, tax_percentage: float) -> float:
        """
        Computes the IVA value based on the amount and tax percentage.
        """
        return amount * (tax_percentage / 100.0)

    @staticmethod
    def compute_total(amount: float, tax_percentage: float) -> float:
        """
        Computes the total amount, which is base amount + IVA.
        """
        return amount + Tax.compute_iva(amount, tax_percentage)

File: my-fresh-copy/hw/basurto/u2/hw09Libraries/JavaScript/TaxLib/utils/tax.js
function computeIva(amount, taxPercentage) {
    return amount * taxPercentage / 100;
}

function computeTotal(amount, taxPercentage) {
    return amount + computeIva(amount, taxPercentage);
}

module.exports = {
    computeIva,
    computeTotal
};